_P172_PZEM-004Tv3.ino

```diff
@@ -0,0 +1,873 @@
+//#define USES_P172
+#ifdef USES_P172
+//#######################################################################################################
+//################### Plugin 172 PZEM-004Tv3 AC Current and Voltage measurement sensor ##################
+//#######################################################################################################
+/*
+    Needed modifications in ESPEasy source code
+        In _CPlugin_SensorTypeHelper.ino:
+        Insert at line 30 and 31
+                case SENSOR_TYPE_SEX:
+                    return 6;
+
+        In ESPEasy_checks.ino:
+        Change line 28:
+        check_size<ExtraTaskSettingsStruct,               472u>();
+        to:
+        check_size<ExtraTaskSettingsStruct,               640u>();
+
+        In DeviceStruct.h:
+        Insert at line 30:
+        #define SENSOR_TYPE_SEX                    66 
+
+        In Custom.h:
+        Add line:
+        #define VARS_PER_TASK 6

```

Comment:
This is something I will not merge without some very very big warnings.
Changing `VARS_PER_TASK` will render all existing settings incompatible.
It will also prevent ESPEasy p2p to work correct (may even crash other nodes in the network) and it is never ever tested.
Things I can imagine that may fail:
- Rules
- Some commands
- Some controllers
- Some other plugins may cause crashes

It is really a very bad idea to change this define.